README.md

```diff
@@ -104,53 +104,50 @@ $config['cas.settings']['proxy']['initialize'] = TRUE;
 
 See the [Cas module](https://www.drupal.org/project/cas) for more information.
 
-## Development
-
-The OpenEuropa Authentication project contains all the necessary code and tools for an effective development process,
-such as:
+## Development setup
```

Comment:
As far as we renaming the subsection of Readme file, we have to update also links in the menu.